app/models/submission.rb

```diff
@@ -133,12 +135,6 @@ def reviewed_by_user
     nil
   end
 
-  def artist
```

Comment:
I don't think we can remove this, otherwise the submission edit page's artist field will appear blank when and admin is editing it.

Reply:
You are right. Moved Gravql call to submissions_controller `before_action` for `edit` and `show`. Now it will appear in edit page too.